features/old/accounts/service_contracts.feature

```diff
@@ -39,22 +39,19 @@ Feature: Account service plans management
      And I should see "Fancy API"
 
     When I follow "Subscribe to Fancy API"
-     And I wait until all animations have finished
```

Comment:
Nice removal, this is not something we should indicate to the user.
It carries no useful information 🎖